Source/WebKit/Scripts/process-entitlements.sh

```diff
@@ -1,5 +1,4 @@
 #!/bin/bash
-set -e
```

Comment:
I wonder if this is required to catch some errors. Is it possible to reset this before running the new script, or have the new script always succeed?

Reply:
I'll restore this.